conceptual/fabrics/fabrics-advising.md

```diff
@@ -4,7 +4,23 @@ uid: fabrics-advising
 
 # Advising the Current Type
 
-Instead of using aspects, you can advice the current type using a type fabric. A type fabric is a compile-time nested class that acts as a type-level aspect added to the nesting type.
+<!---  I'm at something of a loss as to what to suggest here.
```

Comment:
Generally I'm using the AOP terminology that was set in the 1990s and even before (LISP). We say "an advice, the advice" for singular but just "advice" for plural.

Reply:
I will fix in a different commit.